api/api/test/test_validator.py

```diff
@@ -87,7 +87,8 @@
     ('wazuh 4.4.0', _wazuh_version),
     ('wazuh v4.4.0', _wazuh_version),
     # miscellaneous
-    ('aHR0cHM6Ly9zdGFja2FidXNlLmNvbS90YWcvamF2YS8=', _base64)
+    ('aHR0cHM6Ly9zdGFja2FidXNlLmNvbS90YWcvamF2YS8=', _base64),
+    ('correct.wpk', _wpk_path)
```

Comment:
The test case could be under the `#paths`  comment, like the `('local_rules1.xml,local_rules2.xml', _xml_filename)` case to maintain the organization of the test cases

Reply:
Done.